src/dmd/dinifile.d

```diff
@@ -153,7 +153,7 @@ void updateRealEnvironment(StringTable* environment)
         memcpy(s + namelen + 1, value, valuelen);
         s[namelen + 1 + valuelen] = 0;
         //printf("envput('%s')\n", s);
-        putenv(s);
```

Comment:
TODO: Not sure why this call to `putenv` exists...we should determine why this was put here.